src/components/contactUs.css

```diff
@@ -0,0 +1,49 @@
+input[type='text'],
+textarea,
+input[type='email'] {
+  width: 100%;
+  padding-left: 2%;
+  border: none;
+  background-color: #ffffff;
+  border-radius: 10px;
+  resize: none;
+  color: #3f3d56;
+}
+input[type='text'],
+input[type='email'] {
+  height: 82%;
```

Comment:
* Heights are usually used with fixed sizes (vw, vh, px) yet not with percentages in this case because it ruins the design by changing the height of the browser. Open the inspection window in the lower half of your browser and you'll understand me 

Same goes for all the component